### PR TITLE
Poa

### PR DESCRIPTION
**Pull request by sjackman**

### CODE REVIEW

**Comment by sjackman** [comment]
```
Error Message

failed: brew audit poa --strict
Stacktrace

        ==> brew style poa

1 file inspected, no offenses detected
Error: 1 problems in 1 formulae

==> audit problems
poa:
 * Use `system "#{bin}/poa", "2>&1", "|grep", "poa"` instead of `system "#{bin}/poa 2>&1 |grep poa"` 
```

File: poa.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    system "make", "poa"
+    bin.install "poa", "make_pscores.pl"
+    doc.install "README"
+    lib.install "liblpo.a"
+    prefix.install "blosum80.mat", "blosum80_trunc.mat", "multidom.pscore", "multidom.seq"
+    (include/"poa").install Dir["*.h"]
+  end
+
+  test do
+    system "#{bin}/poa 2>&1 |grep poa"
```
Comment (xu-cheng) [inline poa.rb]:
```ruby
assert_match "poa", shell_output("#{bin}/poa 2>&1")
```

**Comment by sjackman** [comment]
@mikemcquaid How do I add a bottle for a new formula?

This bottle job failed: http://bot.brew.sh/job/Homebrew%20Bottles/16014/console

```
==> Applying patch
Applying: poa 2.0: New formula
==> Patch changed:
 poa.rb | 22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)
Error: No available formula for poa 
Build step 'Execute shell' marked build as failure
```

File: poa.rb (hunk 1 above)
Comment (sjackman) [inline poa.rb]: Thanks, Xu.

Comment (sjackman) [inline poa.rb]:
```ruby
assert_match "poa", shell_output("#{bin}/poa 2>&1; true")
```

Comment (xu-cheng) [inline poa.rb]: If it's such case, it should be:
```ruby
assert_match "poa", shell_output("#{bin}/poa 2>&1", expected_exit_code)
```

Comment (sjackman) [inline poa.rb]: Ah, thanks again.

**Comment by sjackman** [comment]
Yosemite and Mavericks were fine, but Mountain Lion failed with an SHA1 mismatch.

http://bot.brew.sh/job/Homebrew%20Science%20Pull%20Requests/1102/version=mountain_lion/testReport/junit/brew-test-bot/mountain_lion/fetch_poa/
```
brew-test-bot.mountain_lion.fetch poa

Failing for the past 1 build (Since Failed#1102 )
Took 2.5 sec.
Error Message

failed: brew fetch --retry --build-bottle --force poa
Stacktrace

        ==> Fetching poa from Homebrew/homebrew-science
==> Downloading https://downloads.sourceforge.net/project/poamsa/poamsa/2.0/poaV2.tar.gz
Downloaded to: /Library/Caches/Homebrew/poa-2.0.tar.gz
SHA1: a2fd3379ea944f6b2f97fb0f79b7b85cb3e14d0b
SHA256: 4c9af4437123ca1d65498a00b70a7a186972a21f14c8b3cc905b9f5d1705142c
==> Retrying download
==> Downloading https://downloads.sourceforge.net/project/poamsa/poamsa/2.0/poaV2.tar.gz
Warning: Formula reports different SHA1: 57f863956736498dd76afb57967036bf92c6d599
Downloaded to: /Library/Caches/Homebrew/poa-2.0.tar.gz
SHA1: a2fd3379ea944f6b2f97fb0f79b7b85cb3e14d0b
SHA256: 4c9af4437123ca1d65498a00b70a7a186972a21f14c8b3cc905b9f5d1705142c
```

**Comment by mikemcquaid** [comment]
This should be fixed as-of https://github.com/Homebrew/homebrew/commit/be5dafdc363454cace3bab286a14f64946f28b9e.

**Pull request closed without merging**